src/Controls/tests/Xaml.UnitTests/GlobalXmlnsWithXStatic.xaml.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Controls;
+using NUnit.Framework;
+
+[assembly: XmlnsDefinition("http://schemas.microsoft.com/dotnet/maui/global", "Microsoft.Maui.Controls.Xaml.UnitTests.NSGlobalXmlnsWithXStatic")]
```

Comment:
I'm a little confused now that I see the usage of this attribute. Do we expect the first parameter to be `"http://schemas.microsoft.com/dotnet/maui/global"` in most cases? If yes, we could simplify this by having something like `[assembly: GlobalXmlnsDefinition("MyNamespace")]`?